packages/p2p/src/components/modal-manager/modals/filter-modal/filter-modal-body.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { Text, Icon, ThemedScrollbars, ToggleSwitch } from '@deriv/components';
+import { observer } from '@deriv/stores';
+import { Localize } from 'Components/i18next';
+import { useStores } from 'Stores';
+import { TPaymentMethod } from './filter-modal';
+import FilterModalResult from './filter-modal-result';
+import FilterModalSearch from './filter-modal-search';
+
+type TFilterModalBodyProps = {
+    onChange: (payment_method: TPaymentMethod) => void;
+    selected_methods: string[];
+    selected_methods_text: string[];
+    setHasMadeChanges: (has_made_changes: boolean) => void;
+};
+
+const FilterModalBody = ({
+    onChange,
+    selected_methods,
+    selected_methods_text,
+    setHasMadeChanges,
+}: TFilterModalBodyProps) => {
+    const { buy_sell_store, my_profile_store } = useStores();
+    const {
+        setShowFilterPaymentMethods,
+        setShouldUseClientLimits,
+        show_filter_payment_methods,
+        should_use_client_limits,
+    } = buy_sell_store;
+    const { payment_methods_list_items } = my_profile_store;
+
+    return (
+        <React.Fragment>
+            {show_filter_payment_methods ? (
+                <React.Fragment>
+                    <FilterModalSearch />
+                    <div className='filter-modal__checkbox-container'>
```

Comment:
Please rename the classnames to use the component name.
```suggestion
                    <div className='filter-modal-body__checkbox-container'>
```

Reply:
do we need a separate css file as well? didn't change it since i didn't create separate files for these.

Reply:
okay then we can proceed with this